backend/src/RedisDriver.js

```diff
@@ -0,0 +1,29 @@
+const Redis = require( 'ioredis' );
+const redisAdapter = require( 'socket.io-redis' );
+const dotenv = require( 'dotenv' );
+dotenv.config( { path: '.env' } );
+
+class RedisDriver {
+	constructor( io ) {
+		this.io = io;
+		this._config = {
+			host: process.env.HOST,
```

Comment:
Environment variable names aren't consistent. I'd change the HOST variable to REDIS_ADAPTER_HOST.
Or change both to REDIS_HOST and REDIS_PORT.

Reply:
> Or change both to REDIS_HOST and REDIS_PORT.

Sounds better.